src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book() {
+    }
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
```

Comment:
Consider creating an additional constructor that includes the 'id' field. This would allow initializing a Book object with an id, which is useful when converting a ResultSet to a Book object. For example: public Book(Long id, String title, BigDecimal price) { this.id = id; this.title = title; this.price = price; }